src/shared/constants.ts

```diff
@@ -28,6 +28,7 @@ export const documentationUrl: string = isCloud9()
  * - alternative?: codecatalyst/latest/userguide/sign-up-create-resources.html
  */
 export const authHelpUrl = 'https://docs.aws.amazon.com/general/latest/gr/differences-aws_builder_id.html'
+export const ssoAuthHelpUrl = 'https://docs.aws.amazon.com/singlesignon/latest/userguide/howtosignin.html'
```

Comment:
This was unfortunately the best I could find. There apparently isn't a generic documentation page for the device code flow. The only thing I found was for the AWS CLI which would obviously be confusing too.

Reply:
The best reference that I found is the one just above this. 

Some ongoing discussion about potentially a new page.